tests/testthat/test-fims-rcpp.R

```diff
@@ -0,0 +1,19 @@
+test_that("Rcpp interface works for modules", {
+
+  fims <- Rcpp::Module("fims", PACKAGE = "FIMS")
+
+  expect_no_error(parameter <- new(fims$Parameter, .1))
+  expect_no_error(beverton_holt <- new(fims$BevertonHoltRecruitment))
+  expect_no_error(logistic_selectivity <- new(fims$LogisticSelectivity))
+  expect_no_error(ewaa_growth <- new(fims$EWAAgrowth))
+  logistic_selectivity$slope$value <- .7
+  logistic_selectivity$median$value <- 5.0
+
+  expect_equal(logistic_selectivity$slope$value, 0.7)
```

Comment:
question: Is this line necessary for testing that the rcpp interface works for modules?

Reply:
same for `expect_equal(ewaa_growth$ages, 1.0)`

Reply:
On second thought, this seems fine. If we have long running tests in the future that we want to slim down, we could consider taking out some more minimal tests.